automation/vars/uni04delta.yaml

```diff
@@ -0,0 +1,80 @@
+---
+vas:
+  uni04delta:
+    stages:
+      - path: examples/dt/uni04delta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
```

Comment:
can also add timeout here too like other dt/vas

Reply:
done

Reply:
I would increase the timeout beyond 60 seconds.  Maybe 3 minutes?

Reply:
I took the value from uni01aplha but ok, np.

Reply:
done